### DISCUSSION TITLE
2022/11/28/2022-11-28-ZFSBootMenu/

### DISCUSSION THREAD

**Discussion by giscus** [Pinghigh/pinghigh.github.io#2] [Announcements]
# 2022/11/28/2022-11-28-ZFSBootMenu/

zstd 下 zfs 根目录压缩率能上 2x，而且对于 gzip 来说性能更好，但是显然 grub 引导不了，其他的不好用，所以折腾了 ZBM

https://pinghigh.github.io/2022/11/28/2022-11-28-ZFSBootMenu/

**Comment by Pinghigh** [comment]
博客已经迁移了喵，新链接是 https://blog.tibrella.top/post/archlinux-on-zfs-root-with-zbm/index.html 喵
